app/build.gradle

```diff
@@ -4,6 +4,8 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-android-extensions'
 
+apply plugin: 'kotlin-kapt'
+
```

Comment:
My suggestion is to create the new ticket in the backlog to add some static analysis/formatting tool

For example: https://github.com/shyiko/ktlint